packages/shared/src/graphql/search.ts

```diff
@@ -1,5 +1,63 @@
-import { gql } from 'graphql-request';
+import request, { gql } from 'graphql-request';
+import { apiUrl, graphqlUrl } from '../lib/config';
+import { isNullOrUndefined } from '../lib/func';
+import { Connection, RequestQueryParams } from './common';
+import { webappUrl } from '../lib/constants';
+import { Post } from './posts';
 
+export const searchPageUrl = `${webappUrl}search`;
```

Comment:
Not in need of a fix, but maybe we should define a global file with all links?
To avoid duplication and ensure once-off useages